_posts/2024-04-29-pyos-newsletter-may-2024.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: May 2024"
+excerpt: "May has brought two new packages, the first (of many!) pyOpenSci workshops, and several incredible speaking opportunities. Read on to learn more!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-may-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+## <i class="fa-solid fa-seedling"></i> Spring is in the Air
+For those of us in the Northern Hemisphere, spring seems to have finally sprung. It’s a lovely time of year, filled with growth, opportunity, and new beginnings. And for all of us here in the pyOpenSci it’s brought new packages, new opportunities, and new projects that we can’t wait to share with you!
+
+## <i class="fa-solid fa-chalkboard-user"></i> pyOpenSci’s first workshop was a success!
+<figure>
+    <a href="/images/blog/2024/may/april-workshop.png">
+    <img src="/images/blog/2024/may/april-workshop.png" style="max-width:100%" alt="Purple line art of a robot standing in a field of solid purple flowers. The text reads 'Build Your First Python Package! a live, online workshop with pyOpenSci. Thursday, April 25th, 2024.">
```

Comment:
@kierisi i just want to test something here with CI. i'll undo it if it breaks things more. i want to see if the image will render in this pr with relative_url
```suggestion
    <img src="{{ /images/blog/2024/may/april-workshop.png | relative_url }}" style="max-width:100%" alt="Purple line art of a robot standing in a field of solid purple flowers. The text reads 'Build Your First Python Package! a live, online workshop with pyOpenSci. Thursday, April 25th, 2024.">
```